packages/contracts/deploy/40_finalize-managing-dao/20_register-managing-dao-on-dao-registry.ts

```diff
@@ -53,6 +55,35 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   console.log(
     `Registered the (managingDAO: ${managingDAOAddress}) on (DAORegistry: ${daoRegistryAddress}), see (tx: ${registerTx.hash})`
   );
+
+  // Set Metadata for the Managing DAO
+  const managingDaoContract = await ethers.getContractAt(
+    'DAO',
+    managingDAOAddress
+  );
+
+  const metadataCIDPath = await uploadToIPFS(
+    JSON.stringify(MANAGING_DAO_METADATA),
+    network.name
+  );
+
+  const SET_METADATA_PERMISSION = ethers.utils.id('SET_METADATA_PERMISSION');
+
+  await managingDaoContract.grant(
+    managingDAOAddress,
+    deployer,
+    SET_METADATA_PERMISSION
+  );
+
+  await managingDaoContract.setMetadata(
+    ethers.utils.hexlify(ethers.utils.toUtf8Bytes(`ipfs://${metadataCIDPath}`))
+  );
+
+  await managingDaoContract.revoke(
+    managingDAOAddress,
+    deployer,
+    SET_METADATA_PERMISSION
```

Comment:
i would add the grant to `00_gran-permissions.ts`
and revoke to 40_revoke-permission.ts` step,
so we reduce the number of transactions

Reply:
Good point. DONE !